### DISCUSSION TITLE
Phidget 1141

### DISCUSSION THREAD

**Discussion by zihyer** [artisan-roaster-scope/artisan#1396] [General]
Hello everyone.  Has anyone tried and/or had any success connecting a [Phidget 1141](https://www.phidgets.com/?prodid=114)?

Tried adding as a couple different additional devices but I can't seem to get a reading from it.  Phidget control panel reads my 1-100% regulator/burner range as 85-94 kPa (voltage ratio input) and 11.82 - 16v in voltage input mode.

[artisan-settings1141.zip](https://github.com/artisan-roaster-scope/artisan/files/13933063/artisan-settings1141.zip)

**Comment by roasterdave** [comment]
Haven't heard of anyone using the Phidget 1141.  It should work as it is similar to the Phidget 1136 that is used by many to measure exhaust air pressure.  Your extra device type should be "**Phidget HUB IO 0**" and the sensor connected to hub port 0.   You will need a symbolic equation to convert from sensor voltage to pressure.  See the Phidget device's user guide for the appropriate formula.  There is an example configuration using the Phidget 1136 in the [first document here](https://www.carefreebuzzbuzz.com/Learning-About-Coffee/mill-city-roaster-upgrades). Look on page 14 in the Fan Pressure Setup section.

_Please be aware -This device is intended to measure the pressure of "clean, non-condensing air".  Consider your safety if you plan to use it with combustible gasses._
